### PR TITLE
Italian translation update (OJS 2.4.8)

### PR DESCRIPTION
**Pull request by marchitelli**

### CODE REVIEW

File: locale/it_IT/admin.xml [hunk 1]
```diff
@@ -60,9 +60,13 @@
 	<message key="admin.mergeUsers.from.description">Seleziona un utente da fondere sull'account di un altro utente (es. quando qualcuno ha due account da utente). L'account selezionato sarà cancellato e le sue submission, gli incarichi ecc. saranno attribuiti al secondo account.</message>
 	<message key="admin.mergeUsers.allUsers">Tutti gli utenti iscritti</message>
 	<message key="admin.mergeUsers.confirm">Sei sicuro di voler fondere {$oldAccountCount} account selezionato(i) con l'account con il nome utente "{$newUsername}"? I vecchi {$oldAccountCount} account non esisteranno più. Questa azione non è reversibile.</message>
-	<message key="admin.mergeUsers.noneEnrolled">Non ci sono utenti iscritti</message>
+	<message key="admin.mergerUsers.noneEnrolled">Non ci sono utenti iscritti</message>
```
Comment (mtub) [inline locale/it_IT/admin.xml]: key is still named admin.mergeUsers.… please revert this change

File: plugins/generic/dataverse/locale/it_IT/locale.xml [hunk 2]
```diff
@@ -0,0 +1,176 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/generic/dataverse/locale/en_US/locale.xml
```
Comment (mtub) [inline plugins/generic/dataverse/locale/it_IT/locale.xml]: path: .../locale/it_IT/locale.xml (replace en_US)

File: plugins/generic/objectsForReview/locale/it_IT/locale.xml [hunk 3]
```diff
@@ -0,0 +1,654 @@
...
+	<message key="plugins.generic.objectsForReview.editor.objectTypes.update">Aggiorna</message>
+	<message key="plugins.generic.objectsForReview.editor.objectTypes.updateLocales">Update locale data for the above selected objects:</message>
+	<message key="plugins.generic.objectsForReview.editor.objectTypes.install">Installa</message>
+	<message key="plugins.generic.objectsForReview.editor.objectTypes.installObjects">Install the missing review objects - choose the types and the locales and press the install button:</message>
+	<message key="plugins.generic.objectsForReview.editor.objectType.name">Nome</message>
+	<message key="plugins.generic.objectsForReview.editor.objectType.description">Descrizione e istruzioni</message>
+	<message key="plugins.generic.objectsForReview.editor.objectType.metadata">Metadati</message>
+	<message key="plugins.generic.objectsForReview.editor.objectType.preview">Anteprima</message>
+
+	<message key="plugins.generic.objectsForReview.editor.objectMetadata">Review Object metadata</message>
```
Comment (mtub) [inline plugins/generic/objectsForReview/locale/it_IT/locale.xml]: not translated (also below) - could you remove the English lines?

File: plugins/generic/pln/locale/it_IT/locale.xml [hunk 4]
```diff
@@ -0,0 +1,107 @@
...
+	<message key="plugins.generic.pln.error.handler.file.notfound">Il file del versamento non è stato trovato. Potrebbe non essere ancora stato impacchettato oppure il pacchetto potrebbe essere stato rimosso.</message>
+
+	<message key="plugins.generic.pln.depositor.statusupdates">Stati di versamento in corso di aggiornamento.</message>
+	<message key="plugins.generic.pln.depositor.updatedcontent">Contenuti aggiornati in lavorazione.</message>
+	<message key="plugins.generic.pln.depositor.newcontent">Nuovi contenuti in lavorazione.</message>
+	<message key="plugins.generic.pln.depositor.packagingdeposits">Impacchettamento dei contenuti per il versamento.</message>
+	<message key="plugins.generic.pln.depositor.transferringdeposits">Invio del versamento alla PLN in corso.</message>
+
+	<message key="plugins.generic.plngateway.displayName">PLN Gateway</message>
+	<message key="plugins.generic.plngateway.description">Reports PLN Status and recent article titles for the staging server.</message>
```
Comment (mtub) [inline plugins/generic/pln/locale/it_IT/locale.xml]: English…

File: plugins/generic/stopForumSpam/locale/it_IT/locale.xml [hunk 5]
```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * plugins/generic/stopForumSpam/locale/en_US/locale.xml
```
Comment (mtub) [inline plugins/generic/stopForumSpam/locale/it_IT/locale.xml]: replace en_US with it_IT

File: plugins/generic/dataverse/locale/it_IT/locale.xml [hunk 6]
```diff
@@ -0,0 +1,176 @@
...
+  *
+  * Copyright (c) 2013-2016 Simon Fraser University Library
+  * Copyright (c) 2003-2016 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Dataverse plugin localization strings
+  -->
+
+<locale name="it_IT" full_name="Italian">
```
Comment (mtub) [inline plugins/generic/dataverse/locale/it_IT/locale.xml]: Italian -> Italiano

File: plugins/generic/objectsForReview/locale/it_IT/locale.xml [hunk 7]
```diff
@@ -0,0 +1,654 @@
...
+<!DOCTYPE locale SYSTEM "../../../../../lib/pkp/dtd/locale.dtd">
+
+<!--
+  * @file plugins/generic/objectsForReview/locale/it_IT/locale.xml
+  *
+  * Copyright (c) 2013-2015 Simon Fraser University Library
+  * Copyright (c) 2003-2015 John Willinsky
+  * Distributed under the GNU GPL v2. For full terms see the file docs/COPYING.
+  *
+  * Objects for review plugin localization strings for the it_IT (Italian) locale.
```
Comment (mtub) [inline plugins/generic/objectsForReview/locale/it_IT/locale.xml]: Italian -> Italiano

File: plugins/generic/objectsForReview/locale/it_IT/locale.xml [hunk 8]
```diff
@@ -0,0 +1,654 @@
...
+  -->
+ 
+<locale name="it_IT" full_name="Italian">
```
Comment (mtub) [inline plugins/generic/objectsForReview/locale/it_IT/locale.xml]: Italian -> Italiano

**Comment by mtub** [comment]
Thank you so much, @marchitelli - I've commented on a few minor things. Could you update this pull request?

File: locale/it_IT/locale.xml [hunk 9]
```diff
@@ -667,6 +651,9 @@ L'impostare del supporto LOCKSS per Open Journal Systems non deve avvenire fino
 	<message key="about.submissionPreparationChecklist">Lista di controllo per l'invio di una proposta</message>
 	<message key="about.submissionPreparationChecklist.description">Gli autori devono controllare la rispondenza della loro proposta di contributo alle richieste seguenti, pena il rifiuto delle proposte che non aderiscano a queste linee guida.</message>
 	<message key="about.copyrightNotice">Diritti degli autori</message>
+	<message key="manager.setup.copyrightYearBasis">Fissa per default i diritti d'autore dei nuovi articoli su</message>
+	<message key="manager.setup.copyrightYearBasis.Article">L'anno verrà tratto dalla data di pubblicazione dei singoli articoli; consigliato per le riviste che pubblicano in maniera più fluida</message>
+	<message key="manager.setup.copyrightYearBasis.Issue">L'anno verrà tratto dalla data di pubblicazione del fascicolo
```
Comment (mtub) [inline locale/it_IT/locale.xml]: close tag at end of line

File: plugins/generic/pln/locale/it_IT/locale.xml [hunk 10]
```diff
@@ -0,0 +1,107 @@
...
+  * PLN plugin localization strings
+  -->
+ 
+<locale name="it_IT" full_name="Italiano">
+	<message key="plugins.generic.pln">Plugin PKP PLN</message>
+	<message key="plugins.generic.pln.description">Il plugin PKP PLN cura il versamento dei contenuti pubblicati dalla rivista nella PKP Private LOCKSS Network a fini di conservazione a lungo termine.</message>
+	<message key="plugins.generic.pln.manager.setup.description"><![CDATA[<strong>PKP Private LOCKSS Network</strong><br /><br />Leggi la <a href="https://pkp.sfu.ca/pkp-lockss/" target="_blank">panoramica sulla PKP PLN</a> e completa <a href="{$plnPluginURL}">questo semplice modulo</a>. Questa opzione permette l'inclusione della tua rivista nella Public Knowledge Project Private LOCKSS Network (PKP PLN).]]></message>
+	<message key="plugins.generic.pln.enabled">Il plugin PKP PLN è stato abilitato.</message>
+	<message key="plugins.generic.pln.disabled">Il plugin PKP PLN è stato disabilitato.</message>
+	<message key="plugins.generic.pln.settings_page">Plugin PKP PLN - Impostazioni<message>
```
Comment (mtub) [inline plugins/generic/pln/locale/it_IT/locale.xml]: wrong closing at end of line: replace `<message>` with `</message>`

**Comment by marchitelli** [comment]
Closing and replacing
